tests/publiccloud/prepare_tools.pm

```diff
@@ -83,6 +83,14 @@ sub run {
     assert_script_run("touch .config/ipa/config");
     assert_script_run("ipa list");
     assert_script_run("ipa --version");
+
+    # Download and Install Terraform
+    assert_script_run('wget -q ' . get_required_var('TERRAFORM_URL'));
+    assert_script_run('unzip terraform_*');
+    assert_script_run('mv terraform /usr/bin/');
+    assert_script_run('rm terraform*');
```

Comment:
I guess you can just call `unzip terraform_* terraform -d /usr/bin/` instead of these three lines

Reply:
thanks for the hint. done.